src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/clients/FeeService.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.iacaseapi.infrastructure.clients;
+
+import feign.FeignException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.FeeType;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.fee.FeeDto;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.fee.FeeResponse;
+import uk.gov.hmcts.reform.iacaseapi.infrastructure.config.FeesConfiguration;
+import uk.gov.hmcts.reform.iacaseapi.infrastructure.config.FeesConfiguration.LookupReferenceData;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
```

Comment:
I'd prefer if we just followed the pattern elsewhere with a constructor and no annotation required.